Issues/89

AndrewDrupal opened the pull request:
front page sidebar
Add field "Title class" to panel pane. It contain bootstrap class bg-___ to set background fot pane title. + Update feature for this panes.

Second part of update in theme - https://github.com/linnovate/socrates/pull/4

zohar commented:
@AndrewDrupal what does it mean? Will the user need to change the panel configuration when they change the theme?

AndrewDrupal commented:
> @AndrewDrupal what does it mean? Will the user need to change the panel configuration when they change the theme?

No.
I added a field, because not all panels need to set the background for the current design. For some panels, we set the background and we pack the background into a feature (I also updated the feature only for sidebar panels on the main page). Users do not need to configure anything.

If you need to add another panel with a header that has a background, you simply add such a panel and put bg-primary in this field - the header background is get from the bootstrap theme. And there is no need to change something in css.

zohar commented:
Merged, keep up the good work!
